include/swift/Runtime/SwiftDtoa.h

```diff
@@ -93,6 +93,15 @@
  #define SWIFT_DTOA_BINARY16_SUPPORT 1
 #endif
 
+/// Does this platform support _Float16 type?
+#ifndef SWIFT_STDLIB_HAS_FLOAT16_TYPE
+# if defined(FLT16_MAX) && !defined(__wasm__)
+#  define SWIFT_STDLIB_HAS_FLOAT16_TYPE 1
```

Comment:
`SWIFT_STDLIB_HAS_FLOAT16_TYPE` is really misleading, since this has nothing to do with Swift's stdlib supporting Float16 or not. Please rename.